src/pages/sign/signIn/page.css

```diff
@@ -0,0 +1,13 @@
+::-webkit-input-placeholder {
```

Comment:
delete `::-webkit-input-placeholder`, we don't need a custom style, especially for a webkit, we use out of the box an

Reply:
done